cypress/e2e/wip/dtrader/contractsDigitsEvenOddresponsive.cy.js

```diff
@@ -0,0 +1,42 @@
+import '@testing-library/cypress/add-commands'
+
+const stakeAmount = '5'
+
+describe('QATEST-6332 -  Verify contract for Digits', () => {
+  const select = ['even', 'odd']
+  beforeEach(() => {
+    cy.c_login()
+    cy.c_visitResponsive('/', 'small')
+    cy.wait(4000)
+    cy.c_skipPasskeysV2()
+    cy.findByTestId('dt_trading-app-card_real_deriv-trader')
+      .findByRole('button', { name: 'Open' })
+      .click({ force: true })
+  })
+
+  select.forEach((select) => {
+    it(`Should buy ${select == 'even' ? 'Even' : 'Odd'} contract from Even/Odd Trade Type`, () => {
+      const evenOdd = select == 'even' ? true : false
```

Comment:
Makes more sense to use isVar for boolean values:
```suggestion
      const isEvenOdd = select == 'even' ? true : false
```